pkg/quota/admission/clusterresourceoverride/api/register.go

```diff
@@ -0,0 +1,56 @@
+package api
+
+import (
+	"github.com/golang/glog"
+
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	"github.com/openshift/origin/pkg/quota/admission/clusterresourceoverride/api/v1"
+)
+
+var SchemeGroupVersion = unversioned.GroupVersion{Group: "", Version: runtime.APIVersionInternal}
+
+// Adds the list of known types to api.Scheme.
+func AddToScheme(scheme *runtime.Scheme) {
+	scheme.AddKnownTypes(SchemeGroupVersion,
+		&ClusterResourceOverrideConfig{},
+	)
+}
+
+// availableVersions lists all known external versions for this group from most preferred to least preferred
+var availableVersions = []unversioned.GroupVersion{v1.SchemeGroupVersion}
+
+func init() {
```

Comment:
> OK. Is there a reason or is this just convention?

Reason.  If you agree that types should be opt-in enabled (this may be questionable, but its the upstream assertion), then you need a way to depend on the types without enabling them at the same time.  This would allow you to wire this into the dependency chain of something like master start, but have it gated.   As you currently have it structured, that power has been removed.